Backend/AlgorithmsManager.py

```diff
@@ -468,6 +473,64 @@ def get_most_frequent_retweeters(self, username, num_tweets=20, num_retweets=20)
 
         return res
 
+    def get_network_tweets_text(self, id2, username, count, depth):
+        textsStr = ""
+        if id2 > 0 or username is not None:
+            # get bunch of tweets starting with the user of given id
+            n = depth  # goes depth level deep
+            # array of tweet texts
+            texts = []
+            self.append_texts_recursive(texts, count, n, 0, id2, username)
+
+            for text in texts:
+                splittext = text.split(' ')
+                for split in splittext:
+                    if not split.startswith('http'):
+                        textsStr += (split + " ")
+        return textsStr
+
+    def append_texts_recursive(self, texts, count, n, i, id2, username):
+        tweets = self.twitter.getTweetsFromUser(id2, username, count)
+        for text in tweets:
+            texts.append(text)
+        if i < n:
+            followsids = self.twitter.getFollowersID(id2, username, count)
+            for id3 in followsids:
+                screen_name = self.twitter.get_username_from_id(id3)
+                self.append_texts_recursive(texts, n, count, i+1, id3, screen_name)
+
+    def create_wordcloud_image(self, id2=0, username=None, count=20, depth=1, querystr=None, c_time = 15):
+        if querystr is None:
+            querystr = "/wordcloud{}{}".format(id2, username)
```

Comment:
I think this'll fail if someone passes in an id number and a username which starts with a numeric character (0-9), but fixing that corner case can wait until we've gotten the feature demoable

Reply:
Yessir!

Reply:
would you mind fixing this now, since we have confirmation from deploying this branch that the code works for demo purposes?

Reply:
Would you mind fixing this now?